README.md

```diff
@@ -10,3 +10,14 @@ https://github.com/mongodb/docs-java
  1. landing - the front page of all the java docs
  2. reference - the reference site for the current version of the driver
 
+## Build Instructions
+
+After making required changes, run the `publish-docs` script with the version:
+
+```sh
+./publish-docs version
+```
+
+This will initiate a submodule (if necessary) that tracks the *gh-pages* branch
+of the [mongo-java-driver](https://github.com/mongodb/mongo-java-driver). It will
+then build the documents, copy items over as necessary, and push them.
```

Comment:
Nit: Future tense
Nit: [Submodule docs](https://git-scm.com/docs/git-submodule#_commands) seems to prefer "add" to "initiate" but, I could be wrong about this (unfamiliar with feature)
Question/Nit: I think the add command is executed every run of the script, so I think the "if necessary" can be removed?
Nit: Could the type of the `mongo-java-driver` link be specified?
Suggestion: I think these steps could be more clearly communicated as an ordered list

```suggestion
This script performs the following actions in the order in which they are listed:

1. Adds a submodule that that tracks the *gh-pages* branch
   of the [mongo-java-driver](https://github.com/mongodb/mongo-java-driver) repository. 
2. Builds the `hugo` documentation site
3. Pushes the built documentation site to the `gh-pages` branch of the `mongo-java-driver` repository
```